src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -478,7 +478,8 @@ object Read extends Logging {
 
     val adamContext = new ADAMContext(sc)
 
-    val adamRecords: RDD[AlignmentRecord] = adamContext.loadAlignments(filename, projection = None, stringency = ValidationStringency.LENIENT)
+    val adamRecords: RDD[AlignmentRecord] = adamContext.loadAlignments(
+      filename, projection = None, stringency = ValidationStringency.LENIENT).rdd
```

Comment:
Compile error. The loadAlignments [method](https://github.com/bigdatagenomics/adam/blob/2996ee644307c312fd9e8e75c560627a2c21b300/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala#L851) seems to now return an `AlignmentRecordRDD` not an actual RDD